fudge-usage-stat.pl

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env perl
+
+######################################################################
+# fudge-useage-stat.pl
+#
+# Usage stats of fudge implementations, versions and verbs for roast.
+#
+######################################################################
+
+use strict;
+use warnings;
+
+use File::Find;
+use File::Spec::Functions qw(canonpath);
+use Text::Table;
+use Cwd qw(cwd);
+
+my $find_dir = @ARGV ? canonpath(shift) : '.';
+
+if ( scalar(my @a =  <$find_dir/S0[1-9]*>) < 10 ) {
+    die <<"USAGE";
+Usage: $0 [roast directory]
+
+   Intended to run against directory with synopses but not many
```

Comment:
At some point we'll drift away from binding to synopses. Just 'tests' would be good enough.